tldr/hybrid_extractor.py

```
@@ -184,6 +192,7 @@ class HybridExtractor:
     LUA_EXTENSIONS = {".lua"}
     LUAU_EXTENSIONS = {".luau"}
     ELIXIR_EXTENSIONS = {".ex", ".exs"}
+    ZIG_EXTENSIONS = {".zig"}
```

Ruff (0.14.14)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)